LEGO The Lord of the Rings/UniversalDIT_Simple-Default.json

```diff
@@ -0,0 +1,3076 @@
+					"`F12`": "Simple-Default_BrownBG/Keyboard Mouse/F12.png",
+					"`Keypad 0`": "Simple-Default_BrownBG/Keyboard Mouse/0.png",
+					"`Keypad 1`": "Simple-Default_BrownBG/Keyboard Mouse/1.png",
+					"`Keypad 2`": "Simple-Default_BrownBG/Keyboard Mouse/2.png",
+					"`Keypad 3`": "Simple-Default_BrownBG/Keyboard Mouse/3.png",
+					"`Keypad 4`": "Simple-Default_BrownBG/Keyboard Mouse/4.png",
+					"`Keypad 5`": "Simple-Default_BrownBG/Keyboard Mouse/5.png",
+					"`Keypad 6`": "Simple-Default_BrownBG/Keyboard Mouse/6.png",
+					"`Keypad 7`": "Simple-Default_BrownBG/Keyboard Mouse/7.png",
+					"`Keypad 8`": "Simple-Default_BrownBG/Keyboard Mouse/8.png",
+					"`Keypad 9`": "Simple-Default_BrownBG/Keyboard Mouse/9.png",
+					"`Keypad +`": "Simple-Default_BrownBG/Keyboard Mouse/ADD.png",
+					"`Keypad .`": "Simple-Default_BrownBG/Keyboard Mouse/DECIMAL.png",
+					"`Keypad /`": "Simple-Default_BrownBG/Keyboard Mouse/DIVIDE.png",
+					"`Keypad *`": "Simple-Default_BrownBG/Keyboard Mouse/MULTIPLY.png",
+					"`Keypad Enter`": "Simple-Default_BrownBG/Keyboard Mouse/RETURN.png",
+					"`Keypad -`": "Simple-Default_BrownBG/Keyboard Mouse/MINUS.png",
+					"`Keypad =`": "Simple-Default_BrownBG/Keyboard Mouse/EQUALS.png"
+				},
+				"SDL/0/Xbox One S Controller": {
+					"`Button 6`": "Simple-Default_BrownBG\\XBOX ONE\\Back.png",
+					"`Button 0`": "Simple-Default_BrownBG\\XBOX ONE\\Button A.png",
+					"`Button 1`": "Simple-Default_BrownBG\\XBOX ONE\\Button B.png",
+					"`Button 2`": "Simple-Default_BrownBG\\XBOX ONE\\Button X.png",
+					"`Button 3`": "Simple-Default_BrownBG\\XBOX ONE\\Button Y.png",
+					"`Axis 0-`": "Simple-Default_BrownBG\\XBOX ONE\\Left Stick.png",
+					"`Axis 0+`": "Simple-Default_BrownBG\\XBOX ONE\\Left Stick.png",
+					"`Axis 1+`": "Simple-Default_BrownBG\\XBOX ONE\\Left Stick.png",
+					"`Axis 1-`": "Simple-Default_BrownBG\\XBOX ONE\\Left Stick.png",
+					"`Hat 0 E`": "Simple-Default_BrownBG\\XBOX ONE\\Pad.png",
+					"`Hat 0 N`": "Simple-Default_BrownBG\\XBOX ONE\\Pad.png",
+					"`Hat 0 S`": "Simple-Default_BrownBG\\XBOX ONE\\Pad.png",
+					"`Hat 0 W`": "Simple-Default_BrownBG\\XBOX ONE\\Pad.png",
+					"`Axis 2-`": "Simple-Default_BrownBG\\XBOX ONE\\Right Stick.png",
+					"`Axis 2+`": "Simple-Default_BrownBG\\XBOX ONE\\Right Stick.png",
+					"`Axis 3+`": "Simple-Default_BrownBG\\XBOX ONE\\Right Stick.png",
+					"`Axis 3-`": "Simple-Default_BrownBG\\XBOX ONE\\Right Stick.png",
+					"`Button 4`": "Simple-Default_BrownBG\\XBOX ONE\\Shoulder L.png",
+					"`Button 5`": "Simple-Default_BrownBG\\XBOX ONE\\Shoulder R.png",
+					"`Button 7`": "Simple-Default_BrownBG\\XBOX ONE\\Start.png",
+					"`Button 8`": "Simple-Default_BrownBG\\XBOX ONE\\Thumb L.png",
+					"`Button 9`": "Simple-Default_BrownBG\\XBOX ONE\\Thumb R.png",
+					"`Full Axis 4+`": "Simple-Default_BrownBG\\XBOX ONE\\Trigger L.png",
+					"`Full Axis 5+`": "Simple-Default_BrownBG\\XBOX ONE\\Trigger R.png"
+				}
+			}

```

Comment:
in the json the button definitions are missing.

https://github.com/Venomalia/UniversalDynamicInput/wiki/Create-a-Package

Reply:
Are the button definitions not line 1438-1638?

Reply:
good work!
You have to make a custom json for Default_BrownBG, the jsons are updated automatically.
then I can add it.